platform/android/MapboxGLAndroidSDK/src/main/AndroidManifest.xml

```diff
@@ -1,20 +1,18 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          xmlns:tools="http://schemas.android.com/tools"
           package="com.mapbox.mapboxsdk">
 
-    <uses-feature android:glEsVersion="0x00020000" android:required="true" />
-    <uses-feature android:name="android.hardware.wifi" android:required="false" /> <!-- Implied by ACCESS_WIFI_STATE. -->
-    <uses-feature android:name="android.hardware.location.gps" android:required="false"/>
+    <uses-feature
+        android:glEsVersion="0x00020000"
+        android:required="true"/>
+    <uses-feature
+        android:name="android.hardware.wifi"
+        android:required="false"/> <!-- Implied by ACCESS_WIFI_STATE. -->
+    <uses-feature
+        android:name="android.hardware.location.gps"
```

Comment:
Can we remove this entry from the manifest? I think it should come with the telemetry library.